integrations/sms/config.json

```diff
@@ -0,0 +1,47 @@
+{
+    "display_name": "SMS",
+    "description": "Send Service alerts via SMS.",
+    "integration_type": "event_output",
+    "supported_event_types": [],
+    "required_fields": [],
+    "max_send_retries": 5,
+    "polling_enabled": false,
+    "test_connection_enabled": true,
+    "parameters": [
+        {
+            "value": "to_phone",
+            "label": "TO (phone number)",
+            "type": "string",
+            "required": true,
+            "help_text": "A phone number that will receive the SMS"
+        },
+        {
+            "value": "from_phone",
+            "label": "FROM (phone number)",
+            "type": "string",
+            "required": true,
+            "help_text": "All SMS alerts will be from this phone number"
+        },
+        {
+            "value": "twilio_account_sid",
+            "label": "Twilio Account SID",
+            "type": "string",
+            "required": true,
+            "help_text": "The \"Twilio Account SID\" - get this from your Twilio account"
```

Comment:
Looks nicer to use the opposite parantheses than to escape same-type parantheses

Reply:
agreed, just need to make sure it's valid JSON

Reply:
@omercnet is right, and since this is json I'm using " exclusively.